docs/design/indexed-job.md

```diff
@@ -874,24 +874,24 @@ admission time; it will need to understand indexes.
 previous container failures.
 - modify the job template, affecting all indexes.
 
-#### Comparison to PetSets
+#### Comparison to StatefulSets
```

Comment:
We should reference that this feature was called PetSet.  Otherwise, people are lacking context and may see this as a new feature.